<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMessage</class>
 <widget class="QDialog" name="SendMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>725</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Create and send message</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/resources.qrc">
    <normaloff>:/icons/48x48/datovka-message-reply.png</normaloff>:/icons/48x48/datovka-message-reply.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoText">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Please fill in at least the &lt;b&gt;subject&lt;/b&gt;, &lt;b&gt;one recipient&lt;/b&gt; and &lt;b&gt;one attachment&lt;/b&gt;:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QLabel" name="fromLabel">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fromUser">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="subjectLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Subject:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="subjectText">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Enter subject of the message</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="replyLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>This is a PDZ prepaid reply. It means that PDZ is paid by recipient.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="optionalFieldCheckBox">
     <property name="text">
      <string>Show optional fields</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="OptionalWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mandate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLineEdit" name="dmLegalTitleLaw"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dmLegalTitleYear"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ยง</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dmLegalTitleSect"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>paragraph</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dmLegalTitlePar"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>letter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dmLegalTitlePoint"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labeldmRecipientRefNumber">
        <property name="text">
         <string>Our reference number:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dmRecipientRefNumber"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Our file mark:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="dmRecipientIdent"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labeldmSenderRefNumber">
        <property name="text">
         <string>Your reference number:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="dmSenderRefNumber"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Your file mark:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="dmSenderIdent"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Personal delivery:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="dmPersonalDelivery">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>To hands:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="dmToHands"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="recipientLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Recipients:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addRecipient">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Add a recipient to the current message</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/3party/plus_16.png</normaloff>:/icons/3party/plus_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeRecipient">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Remove selected recipient from the list</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/3party/delete_16.png</normaloff>:/icons/3party/delete_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findRecipient">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Add recipient from contacts selected from existing messages</string>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/3party/search_16.png</normaloff>:/icons/3party/search_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="recipienthorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="recipientTableWidget">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>1</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>18</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>18</number>
     </attribute>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
      <property name="toolTip">
       <string>Databox ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="toolTip">
       <string>Full name of recipient
or name of organization</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Address</string>
      </property>
      <property name="toolTip">
       <string>Address of recipient</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PDZ</string>
      </property>
      <property name="toolTip">
       <string>The message for recipient will be
sent as Poลกtovnรญ Datovรก Zprรกva</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dmAllowSubstDelivery">
     <property name="toolTip">
      <string>Even if recipient did not read this message,
message is considered as delivered after 
(currently) 10 days. This is delivery through fiction.</string>
     </property>
     <property name="text">
      <string>Allow delivery through fiction</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dmPublishOwnID">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Include sender identification</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="payReply">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Pay transfer charges for reply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="payRecipient">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Use offered payment of transfer charges by recipient</string>
     </property>
     <property name="text">
      <string>Use offered payment of transfer charges by recipient</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_0">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="attachmentLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Attachments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addAttachment">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Add a new file to the attachments</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/3party/plus_16.png</normaloff>:/icons/3party/plus_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeAttachment">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Remove the selected file from attachments</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/3party/delete_16.png</normaloff>:/icons/3party/delete_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openAttachment">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Open selected file in associated application</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/3party/folder_16.png</normaloff>:/icons/3party/folder_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="attachhorizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="attachmentWarning">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Attachment size is larger than 10 MB. Message cannot be sent!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="attachmentTableWidget">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>1</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>18</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>18</number>
     </attribute>
     <column>
      <property name="text">
       <string>File</string>
      </property>
      <property name="toolTip">
       <string>File name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MimeType</string>
      </property>
      <property name="toolTip">
       <string>MIME type of file</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
      <property name="toolTip">
       <string>Size of file in bytes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FilePath</string>
      </property>
      <property name="toolTip">
       <string>Absolute path of file</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Send message</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
